### PR TITLE
Rework NWChem targets and how globbing detects NWChem inputs

### PR DESCRIPTION
**Pull request by zjp**
This patch is intended to get NWChem output files properly imported, and it also picks a more reliable (less verbose) target for reading the relevant data.

### CODE REVIEW

File: NWChem.py [hunk 1]
```diff
@@ -167,11 +167,11 @@ def GetPrerunNMRCalcs(Isomers):
...
     jobdir = os.getcwd()
     os.chdir('nmr')
-    """
+    
     for iso in Isomers:
-        iso.NMRInputFiles = glob.glob(iso.BaseName + 'ginp*com')
-        iso.NMROutputFiles.extend([x[:-4] + '.out' for x in iso.NMRInputFiles if IsGausCompleted(x[:-4] + '.out')])
-    """
+        iso.NMRInputFiles = sorted(glob.glob(iso.BaseName + '*.nw'))
+        iso.NMROutputFiles = sorted(glob.glob(iso.BaseName + '*.nwo'))
```
Comment (KristapsE) [inline NWChem.py:173]: I think we should still only include outputfiles that are valid (ie, have True return value from IsNWChemCompleted.)

**Comment by KristapsE** [comment]
Yeah, these clearly are bugs on my part from working on a copy-pasted code from the Gaussian source file, good spot!

**Comment by zjp** [comment]
Thank you! I'll make the requested change and push it after testing.

File: NWChem.py (hunk 1 above)
Comment (zjp) [reply NWChem.py]: I have reverted this line.

**Comment by KristapsE** [comment]
Thanks for this. I'll go through this and other your pull requests next week, things should get slightly calmer then.

**Pull request closed without merging**